### PR TITLE
[unit-tests-only] Added unit tests todo items for textUtils helper

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
- added todo unit tests for `textUtils` helper from `packages/web-app-files/helper`

## Related Issue

- Part of #5236 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/tests/unit/helpers/textUtils.spec.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+describe('textUtils', () => {
+  // textUtils compares two strings to provide a natural sort
+  // say `a` is the first string and `b` is the second
+  it.todo('should return negative integer if "b" comes before "a"')
+  it.todo('should return positive integer if a comes before "b"')
```
Comment (saw-jan) [inline packages/web-app-files/tests/unit/helpers/textUtils.spec.js:5]:
```suggestion
  it.todo('should return positive integer if "a" comes before "b"')
```

File: packages/web-app-files/tests/unit/helpers/textUtils.spec.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+describe('textUtils', () => {
+  // textUtils compares two strings to provide a natural sort
+  // say `a` is the first string and `b` is the second
+  it.todo('should return negative integer if "b" comes before "a"')
+  it.todo('should return positive integer if "a" comes before "b"')
```
Comment (pascalwengerter) [inline packages/web-app-files/tests/unit/helpers/textUtils.spec.js:5]: I would prefer this to test some more variations, e.g.
- "aa" vs "ab"
- "abb" vs "aba"
- "0" vs "1"
- ...

**Review by pascalwengerter — CHANGES_REQUESTED** [review]

File: packages/web-app-files/tests/unit/helpers/textUtils.spec.js (hunk 2 above)
Comment (kiranparajuli589) [reply packages/web-app-files/tests/unit/helpers/textUtils.spec.js]: added some variations for each cases

Comment (kiranparajuli589) [reply packages/web-app-files/tests/unit/helpers/textUtils.spec.js:22]: actually `.each` cannot be joined with the `.todo` method so I've commented the variations out

**Review by pascalwengerter — APPROVED** [review]

**Pull request merged**